programming-in-scala/src/test/scala/org/programingscala/chapter06/RationalSpec.scala

```diff
@@ -0,0 +1,18 @@
+package org.programingscala.chapter06
+
+import org.scalatest.flatspec.AnyFlatSpec
+
+//people understand how to use your code by looking at your test
+
+class RationalSpec extends AnyFlatSpec {
+
+    "Rational" should "add two positive Rational numbers" in{
+        val oneHalf = new Rational(1, 2)
```

Comment:
will this compile, because the `Rational`  is defined inside `object Chapter06`?

Reply:
> will this compile, because the `Rational` is defined inside `object Chapter06`?

i think this is the last commit, i just pushed it again and it all works now